client/src/charts/NivoCharts.js

```diff
@@ -608,6 +614,119 @@ NivoResponsiveLine.defaultProps = {
   motion_stiffness: 100,
 };
 
+const BubbleNode = ({ node, style, handlers, theme }) => {
+  if (style.r <= 0) return null;
+
+  const min_node_radius = 2;
+
+  const real_r = node.data.isOuter ?
+    style.r :
+    style.r*node.data.ratio > min_node_radius ?
+      style.r*node.data.ratio :
+      min_node_radius;
+  
+  return (
+    <g transform={`translate(${style.x},${style.y})`}>
+      <circle
+        r={real_r}
+        {...handlers}
+        fill={style.fill ? style.fill : style.color}
+        stroke={style.borderColor}
+        strokeWidth={style.borderWidth}
+      />
+    </g>
+  );
+};
+
+
+export class NivoResponsiveBubble extends React.Component{
+  render(){
+    const{
+      margin,
+      is_money,
+      legends,
+      text_formatter,
+      labelSkipWidth,
+      height,
+      value,
+      name,
+      totalValue,
+      totalName,
```

Comment:
I'd put forward these prop names as more meaningful, thoughts?
```
child_value,
child_name,
parent_value,
parent_name,
```

Reply:
Yeah sure, but I want to make the point that this is set up as a visualization for 1 parent and 1 child only, and not a generic parent-child chart.

I might add a comment stating as such just to make sure someone doesn't think it's our generic wrapper around the Nivo bubble chart.

Reply:
Or maybe rename the whole component?